src/Components/Admin/navbar.js

```diff
@@ -118,6 +119,12 @@ export default function Navbar({ home, search, color, roomName }) {
         <div className={classes.root}>
             <AppBar position="fixed" color={color}>
                 <Toolbar className={classes.toolbar}>
+                    <img
+                        src={home ? HoudiniLogoBlack : HoudiniLogoRed}
```

Comment:
Conditional render based on page it is on. The env editor page has "home" boolean set to true.